MonoGame.Framework/Graphics/States/FillMode.cs

```diff
@@ -1,8 +1,21 @@
-﻿namespace Microsoft.Xna.Framework.Graphics
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace Microsoft.Xna.Framework.Graphics
 {
+    /// <summary>
+    /// Defines options for filling the primitive.
+    /// </summary>
     public enum FillMode
     {
+        /// <summary>
+        /// Draw solid faces for each primitive.
+        /// </summary>
         Solid,
+        /// <summary>
+        /// Draw lines connecting the vertices that define a primitive face.

```

Comment:
This line is identical to the XNA docs on MSDN.  We cannot copy Microsoft's docs...  this is the same as copying their code... it is a copyright violation.